### PR TITLE
Added PlayerLoomPatternSelectEvent

### PR DESCRIPTION
**Pull request by Machine-Maker**

### CODE REVIEW

File: Spigot-Server-Patches/0602-Added-PlayerLoomPatternSelectEvent.patch [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
++                return false;
++            }
++            for (EnumBannerPatternType nms : EnumBannerPatternType.values()) {
++                if (event.getPatternType().getIdentifier().equals(nms.getIdentifier())) {
++                    enumBannerPatternTypeOrdinal = nms.ordinal();
++                    break;
++                }
++            }
++            ((Player) entityhuman.getBukkitEntity()).updateInventory();
++            this.d.set(enumBannerPatternTypeOrdinal);
```
Comment (MiniDigger) [inline Spigot-Server-Patches/0602-Added-PlayerLoomPatternSelectEvent.patch]: can we get an obf helper here?

**Review by MiniDigger — DISMISSED** [review]

File: Spigot-Server-Patches/0602-Added-PlayerLoomPatternSelectEvent.patch (hunk 1 above)
Comment (Machine-Maker) [reply Spigot-Server-Patches/0602-Added-PlayerLoomPatternSelectEvent.patch]: Done

**Review by MiniDigger — DISMISSED** [review]

**Review by zachbr — DISMISSED** [review]

**Comment by MiniDigger** [comment]
@limbo-app ready

**Review by limbo-app — APPROVED** [review]
Pull request automatically approved for merging - a core team member has previously approved this pull request.

**Pull request merged**